src/main/java/org/isf/security/jwt/JWTFilter.java

```diff
@@ -0,0 +1,48 @@
+package org.isf.security.jwt;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.util.StringUtils;
+import org.springframework.web.filter.GenericFilterBean;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
```

Comment:
imports in wrong order; see in this repo `.ide-settings\eclipse\OpenHosptital,importorder`.   The order is:

1. static
2. java
3. javax
4. org
5. com
6. everything else